easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb

```
@@ -0,0 +1,31 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-90.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'iperf'
+version = '2.1.9'
+
+homepage = 'http://iperf2.sourceforge.net/'
+description = """Iperf-2.1.9: TCP and UDP bandwidth performance measurement tool"""
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/iperf2/files', 'download')]
+checksums = ['5c0771aab00ef14520013aef01675977816e23bb8f5d9fde016f90eb2f1be788']
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

we should have a `binutils` builddependency when compiling with the `GCCcore` toolchain

style nitpick: lets also move the toolchain paramter up to go between description and sources, for consistency with other easyconfigs

```suggestion
homepage = 'http://iperf2.sourceforge.net/'
description = """Iperf-2.1.9: TCP and UDP bandwidth performance measurement tool"""

toolchain = {'name': 'GCCcore', 'version': '10.2.0'}

sources = [SOURCELOWER_TAR_GZ]
source_urls = [('http://sourceforge.net/projects/iperf2/files', 'download')]
checksums = ['5c0771aab00ef14520013aef01675977816e23bb8f5d9fde016f90eb2f1be788']

builddependencies = [('binutils', '2.35')]
```

---

Thank you for fixes.